services/director-v2/tests/unit/with_dbs/test_route_clusters.py

```diff
@@ -279,14 +284,24 @@ async def test_create_cluster(
         type=random.choice(list(ClusterType)),
     )
     response = await async_client.post(
-        create_cluster_url, json=cluster_data.dict(by_alias=True, exclude_unset=True)
+        create_cluster_url,
+        json=json.loads(
+            cluster_data.json(
+                by_alias=True,
+                exclude_unset=True,
+                encoder=create_json_encoder_wo_secrets(ClusterCreate),
```

Comment:
TIP: for the tests you can create a new class as ``ClusterCreate_ForTests`` or ``ClustersWithoutSecreatesCreate`` that incorporates [the encoder in the ``Config``](https://pydantic-docs.helpmanual.io/usage/exporting_models/#json_encoders) ... then you might use that and avoid lots of